app/src/main/java/com/framgia/fgoal/screen/managerequest/ManagementRequestActivity.java

```diff
@@ -4,13 +4,50 @@
 import android.content.Intent;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
+import android.support.v4.widget.SwipeRefreshLayout;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import android.support.v7.widget.Toolbar;
 import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+import android.widget.Toast;
 
+import com.framgia.fgoal.MainApplication;
 import com.framgia.fgoal.R;
+import com.framgia.fgoal.data.model.ManagementRequest;
+import com.framgia.fgoal.data.source.remote.api.error.BaseException;
 import com.framgia.fgoal.screen.BaseActivity;
+import com.framgia.fgoal.screen.BaseRecyclerViewAdapter;
+import com.framgia.fgoal.screen.managerequest.adapter.ManagementRequestAdapter;
+import com.framgia.fgoal.screen.timeline.EndlessScrollListener;
 
-public class ManagementRequestActivity extends BaseActivity {
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+public class ManagementRequestActivity extends BaseActivity implements ManagementRequestContract.View,
+        AdapterView.OnItemSelectedListener,
+        SwipeRefreshLayout.OnRefreshListener, BaseRecyclerViewAdapter.OnItemClickListener<ManagementRequest> {
+
+    private static final int PROGRESS_DEFAULT = -1;
+    private static final int PROGRESS_LOAD = 1;
+    private static final int PROGRESS_LOAD_MORE = 2;
+    private static final int PROGRESS_REFRESH = 3;
+
+    @Inject
+    ManagementRequestContract.Presenter mPresenter;
+
+    private SwipeRefreshLayout mSwipeRefreshLayout;
+    private ProgressBar mProgressBar;
+    private ProgressBar mProgressBarLoadMore;
+    private RecyclerView mRecyclerManagementRequest;
+    private TextView mTextNoData;
+    private ManagementRequestAdapter mRequestAdapter;
+    private int mLoadData;
```

Comment:
```suggestion
    private int mLoadingType;
```
a nghĩ đặt tên này clear hơn